bench/benchmarks/dds_update_bucket.js

```diff
@@ -0,0 +1,102 @@
+            if (layer.type === 'fill' && layer.paint) {
+                layer.paint['fill-color'] = {
+                    property: 'level',
+                    stops: [[0, 'white'], [100, 'blue']]
+                };
+            }
+        });
+
+        return runSeries(evented, stylesheet, SAMPLE_COUNT, runTile);
+    });
+    return evented;
+};
+
+function runTile (assets, coordinate, callback) {
+    var url = 'https://a.tiles.mapbox.com/v4/mapbox.mapbox-terrain-v2,mapbox.mapbox-streets-v6/' + coordinate.zoom + '/' + coordinate.row + '/' + coordinate.column + '.vector.pbf?access_token=' + config.ACCESS_TOKEN;
+
+    var workerTile = getWorkerTile(coordinate, url);
+
+    var actor = {
+        send: function(action, params, sendCallback) {
+            setTimeout(function() {
+                if (action === 'get icons') {
+                    assets.getIcons(params, sendCallback);
+                } else if (action === 'get glyphs') {
+                    assets.getGlyphs(params, sendCallback);
+                } else assert(false);
+            }, 0);
+        }
+    };
+
+    if (!workerTile.data) {
+        // preload data
+        assets.getTile(url, function(err, response) {
+            if (err) throw err;
+            var data = new VT.VectorTile(new Protobuf(new Uint8Array(response)));
+            // Copy the property data from the vector tile features so we
+            // can use it in updateProperties() during the actual benchmark
+            // In the real use case, the updated property data would be
+            // provided by client code and it wouldn't be pre-tiled, so a
+            // custom source would have to figure out how to generate this
+            // tile-specific payload, probably by joining on an id or
+            // something.
+            workerTile.__props = {};
+            for (var layerId in data.layers) {
+                var layer = data.layers[layerId];
+                workerTile.__props[layerId] = [];
+                for (var i = 0; i < layer.length; i++) {
+                    var properties = layer.feature(i).properties;
+                    workerTile.__props[layerId].push(properties);
+                }
+            }
+            workerTile.parse(data, assets.layerFamilies, actor, null, callback);
+        });
+    } else {
+        workerTile.updateProperties(workerTile.__props, assets.layerFamilies, actor, function (err) {
+            if (err) { return callback(err); }
+            callback();
+        });
+    }
+}
+
+var workerTiles = {};
+function getWorkerTile (coordinate, url) {
+    if (!workerTiles[url]) {
+        workerTiles[url] = new WorkerTile({
+            coord: coordinate,
+            zoom: coordinate.zoom,
+            tileSize: 512,
+            overscaling: 1,
+            angle: 0,
+            pitch: 0,
+            showCollisionBoxes: false,
+            source: 'composite',
+            uid: url
+        });
+    }
+    return workerTiles[url];
+}
+
```

Comment:
I definitely agree in principle with the "mimics real-world usage" criterion.  Couple thoughts in this particular case:

`dds_parse_bucket` differs from the `buffer` benchmark only by:
```js
>         stylesheet.layers.forEach(function (layer) {
>             if (layer.type === 'fill' && layer.paint) {
>                 layer.paint['fill-color'] = {
>                     property: 'level',
>                     stops: [[0, 'white'], [100, 'blue']]
>                 };
>             }
>         });
```

Both do basically the same thing: for "the set of coordinates of tiles used in a "flyTo" from the Mapbox SF office to the Mapbox DC office," run `WorkerTile#parse` on each one.

I think it's worth keeping `dds_parse_bucket`, because:
 - by virtue of having a dds property, the `Bucket#addFeature` calls that are a crucial part of `WorkerTile#parse` are forced to actually populate the paint vertex arrays; this happens in real-world usage, and so it's good to have it covered in a benchmark.
 - moreover, although dds is relatively new, I think it's a pretty good bet that it will become _very_ common, especially once it's supported in Studio; thus, arguably, this might ultimately be mimicing real-world usage better than the `buffer` benchmark does.

As for `dds_update_bucket`, it is again pretty similar to the `buffer` benchmark, except that it focuses on `WorkerTile#updateProperties`.  IMO, we should keep this one only if we're going to bring the dynamic-updating VT source into core; otherwise, it probably better belongs in that repo.  ( 💭 I wonder if there's any good way for a third-party repo to leverage the benchmark infrastructure from here...)